elixir_buildpack.config

```diff
@@ -0,0 +1,2 @@
+erlang_version=18.3
+elixir_version=1.4.0
```

Comment:
@katbow @nelsonic we should probably standardise these versions, I think they're also specified in the `mix.exs`